src/main/java/att/grappa/Grappa.java

```diff
@@ -110,9 +110,15 @@
     /*
      * A instance of ExceptionDisplay used with the (public) convenience methods that follow.
      */
-    private static final ExceptionDisplay exceptionDisplay = new ExceptionDisplay(Grappa.PKG_UPLOW
+    private static ExceptionDisplay exceptionDisplay = new AwtExceptionDisplay(Grappa.PKG_UPLOW
```

Comment:
Do you have a non-AWT implementation that you wouldn't mind sharing?

Reply:
We do logging with log4j. Including my Log4j exception displaying implementation in the grappa code, would introduce new jar dependencies. I do not recommend that. Probably as an example in some comment section?

Reply:
You are right, new dependencies should be avoided at this time.